### DISCUSSION TITLE
Can we use strapi for advanced features in the admin dashboard

### DISCUSSION THREAD

**Discussion by aksthelion** [strapi/strapi#7249] [Legacy Topics]
I'm new to strapi and I can see that it has huge potential. I was wondering if it's possible to customise the admin/dashboard itself so that we can get advanced admin features to manage data in the dashboard itself. I'm coming form traditional monolithic CMS where customising dashboard is what we do all the time. 

E.g.
We want to export the content in csv format in dashboard. Is it doable? If yes how?
Or is it the limitation of strapi that it's mainly for generating APIs and that is all it can do?

**Comment by lauriejim** [comment]
Hello! you will be able to create a new plugin that will let you add a custom view in the admin panel.
If you want you can customize the current admin stuff (frontend & backend)
https://strapi.io/documentation/v3.x/concepts/customization.html
After that you will be able to do what you want with the app. Create custom endpoint, export your data, well no limit.
